packages-content-model/roosterjs-content-model-editor/lib/editor/plugins/ContentModelEditPlugin.ts

```diff
@@ -112,13 +116,17 @@ export default class ContentModelEditPlugin implements EditorPlugin {
         }
     }
 
-    private handleKeyDownEvent(editor: IContentModelEditor, rawEvent: KeyboardEvent) {
+    private handleKeyDownEvent(
+        editor: IContentModelEditor,
+        rawEvent: KeyboardEvent,
+        handledByEditFeature: boolean
+    ) {
         const which = rawEvent.which;
 
         if (!this.editWithContentModel || rawEvent.defaultPrevented) {
```

Comment:
You can add the check to handledByEditFeature. We can treat handledByEditFeature in the same way with rawEvent.defaultPrevented